kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/UpsertDeleteValidator.java

```diff
@@ -61,6 +65,13 @@ protected Optional<String> doValidate(BigQuerySinkConfig config) {
       ));
     }
 
+    if (mergeInterval != -1 && mergeInterval < 10_000L) {
+      logger.warn(String.format(
+              "%s should be at least 10 seconds",
+              MERGE_INTERVAL_MS_CONFIG
```

Comment:
% should not be set to less than 10 seconds. A validation would be introduced in a future release to this effect.

Reply:
<img width="1269" alt="Screenshot 2022-12-28 at 3 36 44 PM" src="https://user-images.githubusercontent.com/117655560/209795330-9aea9f98-10e0-4c0a-ac94-2c385abae187.png">